src/mlpack/tests/cli_test.cpp

```diff
@@ -7,6 +7,8 @@
 
 #include <iostream>
 #include <sstream>
+#include <chrono>
```

Comment:
This is a really minor comment but I think that this include is unnecessary because timers.hop includes chrono already.